server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -104,6 +116,10 @@ public Invocations getInvocations() {
             // Cache Manager config
             .invocation().methods(GET).path("/v2/cache-managers/{name}/config").handleWith(this::getConfig)
```

Comment:
Is the plan to deprecate `/v2/cache-managers` and move the additional methods to `v2/container` in a different PR?

I'm fine if that's the case, but we should create a Jira to track this.

Reply:
Yes, it should be separate IMO

Reply:
https://issues.redhat.com/browse/ISPN-13168